source/_posts/Integral-Image.md

```diff
@@ -0,0 +1,102 @@
+---
+title: How to calculate Integral Image or Summed Area Table
+date: 2020-11-19 16:31:19
+author: Anubhav Rohatgi
+category: AI
+tags: 
```

Comment:
Use keywords throughout your article - Keywords may be keyword phrases rather than just single words

Reply:
done

Reply:
done now